particles/Processing/recipes/ImageClassifier.recipes

```diff
@@ -0,0 +1,90 @@
+import '../Infer.particle'
+import '../Dispose.particle'
+import '../ExpandDims.particle'
+import '../ImageToTensor.particle'
+import '../ResizeBilinear.particle'
+import '../LoadModel.particle'
+import '../Normalize.particle'
+import '../ParseLabels.particle'
+import '../Reshape.particle'
+import '../TopLabels.particle'
+import '../ImageSelector.particle'
+import '../PresentLabel.particle'
+import '../ModelSelector.particle'
+
+import './ClassifierStatics.manifest'
+
+recipe ImageClassifierUI
+  // since we `use`, this recipe is incomplete and coalescer is triggered
```

Comment:
This is a really useful fact.

Reply:
I didn't realize that this works in both directions (consuming and producing handles).